src/fontr/fontr/logger.py

```diff
@@ -0,0 +1,31 @@
+from pytorch_lightning.loggers import WandbLogger
+from kedro.config import ConfigLoader, MissingConfigException
+from kedro.framework.project import settings
+import os
+
+class TorchLogger():
+    logger = None
```

Comment:
I don't see this being used

Reply:
Sure, it's clearly static. I asked this question because I did not see any use of this field, in particular initialization. I see you added it in the "Final remarks" commit. Can you squash that commit?